lib/apiservers/engine/backends/image.go

```diff
@@ -184,6 +155,7 @@ func (i *Image) PullImage(ctx context.Context, ref reference.Named, metaHeaders
 		return err
 	}
 
+	layerCache.Update()
```

Comment:
Yeah, I see the problem.  I guess once we have the persistent key/value store, we can optimize this.  Ok, the full update is a must for now.